### ISSUE TITLE
Seems a bug in estimate_pose_points

### ISSUE THREAD

**Issue by Eddylib** [oliver-batchelor/multical#73]
https://github.com/oliver-batchelor/multical/blob/76b8987d4851af44c83cee2a03abacd4fde2d7cf/multical/board/common.py#L42

In this line undistorted points are input to cv2.solvePnPGeneric, the distortion should be zeros: np.zeros(camera.dist.shape).

In my fisheye dataset, the current version,  cv2.solvePnPGeneric leads to average error of 18.69. When np.zeros(camera.dist.shape) are input to  cv2.solvePnPGeneric ,  average error becomes to 3.44

**Comment by oliver-batchelor** [comment]
Yes I think you're absolutely right, this looks like it is undistorting
twice. Possibly I never noticed because I didn't test with high distortion
cameras and it still find an OK initial solution...

**Comment by Eddylib** [comment]
And there might be other issues in code for fisheye data. In my fisheye dataset, the code outputs very large reproject or during optimize, and will stop immediately.

**Comment by oliver-batchelor** [comment]
If you find out why, I'll be happy to accept a pull request. I don't
currently use this tool myself, and the fisheye code was supplied by a 3rd
party so I haven't used it previously!

**Comment by Eddylib** [comment]
Okay, I'll take some time to figure it out. Thank you for your reply and your excellent work.
